testfiles/properties/fits-no-consolidation.xml

```diff
@@ -43,8 +43,8 @@
 	</process>
 	
 	<!-- file name of the droid signature file to use in tools/droid/-->
-	<droid_sigfile>DROID_SignatureFile_V102_Alt.xml</droid_sigfile>
-	<droid_container_sigfile>container-signature-20211216.xml</droid_container_sigfile>
+	<droid_sigfile>DROID_SignatureFile_V109_Alt.xml</droid_sigfile>
+	<droid_container_sigfile>container-signature-20221102.xml</droid_container_sigfile>
```

Comment:
11/02 seems like a downgrade from 12/16... is there a reason?

Reply:
That's 12/16/21 vs 11/02/22